shimmy/meltingpot_compatibility.py

```diff
@@ -176,10 +177,20 @@ def reset(
 
         observations = utils.timestep_to_observations(timestep)
 
-        return observations, {
-            "step-type": timestep.step_type,
-            "discount": timestep.discount,
-        }
+        # duplicate infos
+        info: dict[AgentID, Any] = dict(
```

Comment:
Could we not make this a dict comprehsion
```python
info = {agent: {"timestep.discount": timestep.discount, "timestep.step_type": timestep.step_type} for agent in self.agents}
```

Reply:
I agree that this would be better! For consistency, I just did what was done in the dm control wrapper - https://github.com/Farama-Foundation/Shimmy/blob/abf3122d1c85c8caf6bec41ee0c68bf74ec136e3/shimmy/dm_control_multiagent_compatibility.py#L54.

Reply:
Let's change them both to dict comprehensions as they are much more readable

Reply:
Done in https://github.com/Farama-Foundation/Shimmy/pull/111/commits/85c8cc24d29f281617995ad01bf7d5f3438b7dff.

Reply:
LGTM